spec/spec.tex

```diff
@@ -581,11 +581,11 @@ \subsection{Invalidates}
 For the purposes of simulation, invalidated components are initialized to random values, and operations involving indeterminate values produce undefined behaviour. This is useful for early detection of errors in simulation.
 
 \subsubsection{The Invalidate Algorithm}\label{invalidate_algorithm}
-Invalidating a component with a ground type indicates that the component's value is indetermined if the component has sink or duplex flow (see section \ref{flows}). Otherwise, the component is unaffected.
+Invalidating a component with a ground type indicates that the component's value is undetermined if the component has sink or duplex flow (see section \ref{flows}). Otherwise, the component is unaffected.
```

Comment:
Actually, on second thought, perhaps this should be "indeterminate"?  I am not sure exactly which meaning is intended, so some guidance would be appreciated.

Reply:
I think it should be undetermined.